### ISSUE TITLE
consider providing an `environment.yml` file for conda users

### ISSUE THREAD

**Issue by ctb** [codemeleon/seqPanther#7]
given instructions in README to use conda, might be nice to provide a conda environment.yml file - here's one that I'm trying out now:
```
name: seqPanther
channels:
    - conda-forge
    - bioconda
    - defaults
dependencies:
    - bcftools
    - samtools
    - muscle=3.8.31
    - ucsc-blat
    - mafft
    - pysam
```
will update / confirm when and if it works!

**Comment by ctb** [comment]
I ran:
```
mamba env create -f environment.yml
```
and it completed and created an environment named `seqPanther` 🎉 

I then did:
```
pip install -e .
```
and now it's installing biopython, click, pandas, numpy, matplotlib, and a few others. These could be added to the above environment.yml if you like.

but! either way! seqpanther install works in the sense that I can now execute `seqpanther` 🎉

**Comment by jsan4christ** [comment]
Awesome @ctb, would you consider making a pull request? I am happy to merge that in.

**Comment by jsan4christ** [comment]
This is resolved in commit [5ccc42c](https://github.com/codemeleon/seqPanther/commit/5ccc42c09d3e9087867caeb7351bdcc11d43d7ae).
